src/funcTest/groovy/me/champeau/jmh/AbstractFuncSpec.groovy

```diff
@@ -30,16 +30,24 @@ abstract class AbstractFuncSpec extends Specification {
             GradleVersion.current()
     ]
 
-    protected static final List<String> TESTED_SHADOW_PLUGINS = [
-            'com.github.johnrengelman.shadow',
-            'io.github.goooler.shadow'
+    /** Plugin + min Gradle version the plugin supports. */
+    protected static final Map<String, GradleVersion> TESTED_SHADOW_PLUGINS = [
+            'com.gradleup.shadow':              GradleVersion.version('8.3'),
+            'com.github.johnrengelman.shadow':  GradleVersion.version('8.3')
```

Comment:
Yea - but somehow it complains that `This version of Shadow supports Gradle 8.3+ only. Please upgrade.`.
I suspect, there's a class-collision, because `pluginsUnderTest` specifies both plugins - and those use the same class names.

Reply:
Weird, my fork has the same min Gradle requirement.

Reply:
Yeah, those 2 classes are both in the classpath, and the new Shadow is loaded the first.

I'll try to fix it.

Reply:
c1cfdb20b1d1d56a2e0747746896ad4e6da25864